lib_methods.rb

```diff
@@ -0,0 +1,118 @@
+require_relative 'book'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+
+# rubocop:disable Metrics/ModuleLength
```

Comment:
- [ ] Please kindly fix this Linter problem. Let's not disable them. Linters detect and flag programming errors to make sure our codebase is up to standard.

Reply:
The codebase has been refactored to suite the SOLID principles